### PR TITLE
Remove the deno adapter

### PR DESCRIPTION
**Pull request by matthewp**
## Changes

- The Deno team has taken over maintenance of the adapter, which is now here: https://github.com/denoland/deno-astro-adapter
- Needs https://github.com/withastro/docs/pull/4741

## Testing

N/A

## Docs

N/A

### CODE REVIEW

File: packages/integrations/deno/README.md
Comment (natemoo-re) [inline packages/integrations/deno/README.md:1]: Might we want to keep some note in this README just to point to the new repo?

Comment (matthewp) [reply packages/integrations/deno/README.md:1]: updated

File: packages/integrations/deno/package.json
Comment (ematipico) [inline packages/integrations/deno/package.json:1]: I belive you need to run a `pnpm install` to update the `package-lock.yml` file

**Review by ematipico — APPROVED** [review]
A reminder that when we publish a new version (`major` presumably), we have to update the index file to throw an error with a message saying that the package is deprecated, and moved to the new org.

File: packages/integrations/deno/package.json
Comment (matthewp) [reply packages/integrations/deno/package.json:1]: This isn't moved to a new (npm) org, just a new GitHub org. They might want to move it to their own npm org in the future, however.

**Review by natemoo-re — APPROVED** [review]

File: packages/integrations/deno/README.md [hunk 1]
```diff
@@ -1,176 +1,3 @@
...
-## Contributing
-
-This package is maintained by Astro's Core team. You're welcome to submit an issue or PR!
-
-## Changelog
-
-See [CHANGELOG.md](CHANGELOG.md) for a history of changes to this integration.
-
-[astro-integration]: https://docs.astro.build/en/guides/integrations-guide/
+The Deno adapter has a new home as part of the [Deno organization](https://github.com/denoland/deno-astro-adapter).
```
Comment (sarah11918) [inline packages/integrations/deno/README.md:3]:
```suggestion
This adapter is no longer maintained by Astro. Please see [the new repository for the Deno adapter](https://github.com/withastro/netlify-adapter) which is now maintained by the Deno organization.

[Instructions for removing this adapter and migrating to the new one](https://docs.astro.build/en/guides/integrations-guide/deno/) can be found in the Astro docs.
```
Just making link text as explicit as possible, and linking to our docs page which has instructions for updating your project.

**Review by sarah11918 — APPROVED** [review]
Just a suggestion below, and pointing people to our own docs page if they need to remove/migrated.

**Pull request merged**